csm/core/agent/api.py

```diff
@@ -368,7 +368,7 @@ def run(port: int, https_conf: ConfSection, debug_conf: DebugConf):
             ssl_context = None
 
         web.run_app(CsmRestApi._app, port=port, ssl_context=ssl_context,
-                    access_log=Log.logger, access_log_format=const.REST_ACCESS_FORMAT)
+                    access_log=None, access_log_format=const.REST_ACCESS_FORMAT)
```

Comment:
will it affect auditlog entry too?
Ideally we should filter out ssl polling call and rest all should be entered in audit log.

Reply:
No this will not effect auditlog entry. Currently I am not seeing Usl polling log. Polling is removed from Usl @tb-s can comment on that.

Reply:
started uds service on vm, I can see polling logs now.

Reply:
added code to filter usl audit logs based on csm.conf entry.